### PR TITLE
Adding time series plot to analytics

### PR DESCRIPTION
**Pull request by srina1h**

### CODE REVIEW

File: code/get_analysis.py [hunk 1]
```diff
@@ -73,8 +75,25 @@ def viewRemaining(chat_id, bot):
...
 
+    random_time = time.time()
+    img_name = str(random_time)+".png"
+    plt.savefig(img_name)
+    bot.send_photo(chat_id, photo=open(img_name, 'rb'), reply_markup=types.ReplyKeyboardRemove())
+
+def viewHistory(chat_id, bot):
+    if not helper.getUserHistory(chat_id):
+        bot.send_message(chat_id, "No history available", reply_markup=types.ReplyKeyboardRemove())
+
```
Comment (shashank-madan) [inline code/get_analysis.py:90]: control continues. Have to add return statement

Comment (shashank-madan) [inline code/get_analysis.py:90]:
```suggestion
        bot.send_message(chat_id, "No history available", reply_markup=types.ReplyKeyboardRemove())
            return
```

**Review by shashank-madan — CHANGES_REQUESTED** [review]

File: code/get_analysis.py (hunk 1 above)
Comment (srina1h) [reply code/get_analysis.py:94]: mb added now.

**Review by shashank-madan — APPROVED** [review]

**Comment by shashank-madan** [comment]
Merging with main

**Pull request merged**